code/tyler/django/django_lab-1/todo_app/models.py

```diff
@@ -0,0 +1,28 @@
+from tkinter import CASCADE
+from django.db import models
+from django.forms import CharField
```

Comment:
Be careful with using `tab` to autocomplete. Recent updates to VSCode will perform automatic imports on things. Lines 1 and 3 are irrelevant in this case